_data/retail.yml

```diff
@@ -265,6 +265,13 @@ websites:
     twitter: Massdrop
     img: massdrop.png
 
+  - name: MediaMarkt
```

Comment:
MediaMarkt operates in several countries. Please specify the country in some way. (Germany/Deutschland etc)